.github/ISSUE_TEMPLATE/use_case.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Event Gateway Use Case
+about: Share your use case(s) you expect to be considered during the development of the AsyncAPI Event Gateway.
+title: "[USECASE] "
+labels: 'use case'
+assignees: ''
+
+---
+
+# AsyncAPI Event Gateway Use Case
+
+|Project name|Reviewers|Company|
```

Comment:
Reviewers? 😄 `Interested`, `Applicants` will be better.

Reply:
Reviewers term was in the original template. Imo it refers to the users who are working on the project the use case belongs to.
However, It would be nice if any of @fmvilas or @derberg can clarify just in case.

Reply:
> it refers to the users who are working on the project the use case belongs to

Yup, that's exactly the meaning. It's the people who made the review.